lib/models/per-bundle-addon-cache/cache-entry.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+/**
+ * Class for the entries in the SingletonAddonCache. The cache itself is keyed
```

Comment:
Tweak this comment (we renamed away from the singleton verbiage)

Reply:
Removed the CacheEntry class as not providing a ton of value over plain POJO.